task_14/pageobjects/homepage.js

```diff
@@ -0,0 +1,29 @@
+const {Builder, By, Key, until} = require('selenium-webdriver')
+const BasePage = require('./basepage')
+var webdriver = require('selenium-webdriver')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887247)
```suggestion
var webdriver = require("selenium-webdriver")
```